data/net/minecraft/SharedConstants.mapping

```diff
@@ -100,11 +100,11 @@ CLASS net/minecraft/SharedConstants
 		ARG 0 input
 			COMMENT the input string
 	METHOD getCurrentVersion ()Lnet/minecraft/WorldVersion;
-		COMMENT {@return the {@link WorldVersion world version}}
+		COMMENT @return the {@link WorldVersion world version}
 		COMMENT
 		COMMENT @throws IllegalStateException if a world version has not been set previously
 	METHOD getProtocolVersion ()I
-		COMMENT {@return the networking protocol version in use by this game version}
+		COMMENT @return the networking protocol version in use by this game version
```

Comment:
These doubled braces are intentional, as `{@return}` inline tags are part of the [JDK 17 Standard Doclet Documentation Comment Specification](https://docs.oracle.com/en/java/javase/17/docs/specs/javadoc/doc-comment-spec.html#return).